addons/event_sale/tests/test_event_internals.py

```diff
@@ -16,22 +16,15 @@ def test_event_configuration_from_type(self):
         """ In addition to event test, also test tickets configuration coming
         from event_sale capabilities. """
         event_type = self.event_type_complex.with_user(self.env.user)
-        event_type.write({
-            'use_mail_schedule': False,
-            'use_ticket': False,
-        })
-        self.assertEqual(event_type.event_type_ticket_ids, self.env['event.type.ticket'])
 
         event = self.env['event.event'].create({
             'name': 'Event Update Type',
             'event_type_id': event_type.id,
             'date_begin': FieldsDatetime.to_string(datetime.today() + timedelta(days=1)),
             'date_end': FieldsDatetime.to_string(datetime.today() + timedelta(days=15)),
         })
-        self.assertEqual(event.event_ticket_ids, self.env['event.event.ticket'])
```

Comment:
Would keep this test as this is a default value for an event.

Reply:
The issue here is that previously when "use_tickets" is set to True, a compute method which removes all existing tickets is triggered, this leads to the event_type having no tickets.

Since "use_tickets" has been removed and with it the compute method, this current event type is no longer expected to have no tickets, instead it should have two tickets, see https://github.com/odoo-dev/odoo/blob/1fb12c4db8172f91bdcc5b5cbb91a2a707e55347/addons/event/tests/common.py#L67.

For this reason I don't think that we can keep this test like it is now, instead I think that we should test something else like tickets count for example?

Can you tell me what do you think about it?

Reply:
Well ok. We have enough tests for tickets I think.